src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -30,9 +32,9 @@ public void testCallCheck() {
 	//Fails for implementations based on IntLinkedList, because it can't be full
 	@Test
 	public void testIsFull() {
-		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer(new IntArrayStack());
 		final int STACK_CAPACITY = 12;
-		for (int i = 0; i < STACK_CAPACITY; i++) {
+		for (int i = 0; i < STACK_CAPACITY - 1; i++) {
```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. If STACK_CAPACITY is 12 then 12 elements should be added to the stack.